packages/Caching/Detector/ChangedFilesDetector.php

```diff
@@ -144,12 +145,14 @@ private function resolvePath(string $filePath): string
 
     private function getFilePathCacheKey(string $filePath): string
     {
-        return sha1($this->resolvePath($filePath));
+        $fileHasher = new FileHasher();
```

Comment:
any reason not to make it an injected service?

Reply:
good point, fixed.